addons/mail/wizard/mail_compose_message.py

```diff
@@ -478,7 +478,7 @@ def render_message(self, res_ids):
         if self.template_id:
             template_values = self.generate_email_for_composer(
                 self.template_id.id, res_ids,
-                ['email_to', 'partner_to', 'email_cc', 'attachment_ids', 'mail_server_id'])
+                ['subject', 'body_html', 'email_to', 'partner_to', 'email_cc', 'attachment_ids', 'mail_server_id'])
```

Comment:
Not sure about this one. The body and subject are generated just above and you override it. @tde-banana-odoo is probably more qualified on this.